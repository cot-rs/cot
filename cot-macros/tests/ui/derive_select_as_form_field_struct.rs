use cot_macros::SelectAsFormField;

#[derive(SelectAsFormField)]
struct NotAnEnum {
    x: u8,
}

fn main() {}

