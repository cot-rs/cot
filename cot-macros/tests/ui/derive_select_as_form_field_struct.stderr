error: `SelectAsFormField` can only be derived for enums
 --> tests/ui/derive_select_as_form_field_struct.rs:3:10
  |
3 | #[derive(SelectAsFormField)]
  |          ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `SelectAsFormField` (in Nightly builds, run with -Z macro-backtrace for more info)
